portfolio/src/main/webapp/index.html

```diff
@@ -75,38 +75,40 @@ <h1>Sanjana Jadhav's Portfolio</h1>
                 </p>
                 <fieldset>
                     <p class="p1">My Gallery</p>
-                    <img src="./images/2.png" width="300" height="330" alt="">
-                    <img src="./images/3.png" width="300" height="330" alt="">
-                    <img src="./images/4.png" width="300" height="330" alt="">
+                    <img src="./images/2.png" width="300" height="380" alt="">
+                    <img src="./images/3.png" width="300" height="380" alt="">
```

Comment:
I would perhaps create a class in css[1] and give it the height and width changes so you can reuse it and not add the same height and width parameters every time

[1]https://www.w3schools.com/cssref/sel_class.asp

Reply:
I'll push it to the other pull request branch, so it has the new changes?